HueBoo/Models/ColorSet.swift

```diff
@@ -16,15 +16,10 @@ struct ColorSet: Equatable {
     let alpha: CGFloat
     
     init(initialColor: UIColor) {
-        var hue: CGFloat = 0
-        var saturation: CGFloat = 0
-        var brightness: CGFloat = 0
-        var alpha: CGFloat = 1
+        var hue, saturation, brightness, alpha: CGFloat
+        (hue, saturation, brightness, alpha) = (0,0,0,0)
```

Comment:
You did this in getTextColor() and I thought it was awesome, very "pythonic"!  I thought you might be able to use the same pattern here